dev-docs/bidders/iqx.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: bidder
+title: IQX
+description: IQX Bidder Adapter
+biddercode: iqx
+media_types: banner, video, native
```

Comment:
Question: Do you have any specs for your native assets? I'm just curious, because so far there has been no known documentation of those in the prebid docs. However this format requires more coordination with publisher and SSP, so I'm curious if you could and would provide specs or at least a reference to it in the prebid docs

Reply:
@muuki88 We plan to use this adapter (server side only, pbjs: false) to streamline integration with SSP. This bidder will bid on native, but rendering on the publisher's side will be addressed by the SSP